src/app/zap-templates/zcl/data-model/chip/camera-av-stream-management-cluster.xml

```diff
@@ -0,0 +1,578 @@
+  </struct>
+
+  <struct name="AudioStreamStruct" apiMaturity="provisional">
+    <cluster code="0x0551"/>
+    <item fieldId="0" name="AudioStreamID" type="int16u"/>
+    <item fieldId="1" name="StreamType" type="StreamTypeEnum" min="0x00" max="0x03"/>
+    <item fieldId="2" name="AudioCodec" type="AudioCodecEnum" min="0x00" max="0x01"/>
+    <item fieldId="3" name="ChannelCount" type="int8u"/>
+    <item fieldId="4" name="SampleRate" type="int32u"/>
+    <item fieldId="5" name="BitRate" type="int32u"/>
+    <item fieldId="6" name="BitDepth" type="int8u"/>
+    <item fieldId="7" name="ReferenceCount" type="int8u"/>
+  </struct>
+
+  <struct name="PerStreamStruct" apiMaturity="provisional">
+    <cluster code="0x0551"/>
+    <cluster code="0x0552"/>
+    <item fieldId="0" name="StreamID" type="int16u"/>
+    <item fieldId="1" name="OnOff" type="boolean"/>
+  </struct>
+
+  <struct name="RateDistortionTradeOffPointsStruct" apiMaturity="provisional">
+    <cluster code="0x0551"/>
+    <item fieldId="0" name="Codec" type="VideoCodecEnum" min="0x00" max="0x03"/>
+    <item fieldId="1" name="Resolution" type="VideoResolutionStruct"/>
+    <item fieldId="2" name="MinBitRate" type="int32u"/>
+  </struct>
+
+  <struct name="SnapshotParamsStruct" apiMaturity="provisional">
+    <cluster code="0x0551"/>
+    <item fieldId="0" name="Resolution" type="VideoResolutionStruct"/>
+    <item fieldId="1" name="MaxFrameRate" type="int16u"/>
+    <item fieldId="2" name="ImageCodec" type="ImageCodecEnum" min="0x00" max="0x00"/>
+  </struct>
+
+  <struct name="SnapshotStreamStruct" apiMaturity="provisional">
+    <cluster code="0x0551"/>
+    <item fieldId="0" name="SnapshotStreamID" type="int16u"/>
+    <item fieldId="1" name="ImageCodec" type="ImageCodecEnum" min="0x00" max="0x00"/>
+    <item fieldId="2" name="FrameRate" type="int16u"/>
+    <item fieldId="3" name="BitRate" type="int32u"/>
+    <item fieldId="4" name="MinResolution" type="VideoResolutionStruct"/>
+    <item fieldId="5" name="MaxResolution" type="VideoResolutionStruct"/>
+    <item fieldId="6" name="Quality" type="int8u" min="1" max="100"/>
+    <item fieldId="7" name="ReferenceCount" type="int8u"/>
+  </struct>
+
+  <struct name="VideoResolutionStruct" apiMaturity="provisional">
+    <cluster code="0x0551"/>
+    <item fieldId="0" name="Width" type="int16u"/>
+    <item fieldId="1" name="Height" type="int16u"/>
+  </struct>
+
+  <struct name="VideoSensorParamsStruct" apiMaturity="provisional">
+    <cluster code="0x0551"/>
+    <item fieldId="0" name="SensorWidth" type="int16u"/>
+    <item fieldId="1" name="SensorHeight" type="int16u"/>
+    <item fieldId="2" name="HDRCapable" type="boolean"/>
+    <item fieldId="3" name="MaxFPS" type="int16u"/>
+    <item fieldId="4" name="MaxHDRFPS" type="int16u"/>
+  </struct>
+
+  <struct name="VideoStreamStruct" apiMaturity="provisional">
+    <cluster code="0x0551"/>
+    <item fieldId="0" name="VideoStreamID" type="int16u"/>
```

Comment:
Spec comment: the spec has lots of "MS" in the Default comment, which probably don't make sense.  But since in practice the XML ignores those anyway, it's not a problem for this PR.

Reply:
MS values for these attributes have been removed in the Spec.